bittensor/config.py

```diff
@@ -344,6 +400,18 @@ def _merge(cls, a, b):
                 a[key] = b[key]
         return a
 
+    def deep_merge(self, dict1, dict2):
```

Comment:
If possible refactor these utility functions into a util class or even just import a dependency that does these things.